modules/openapi-generator/src/test/resources/3_0/petstore.json

```diff
@@ -996,6 +999,17 @@
           "name": "Order"
         }
       }
+    },
+    "headers": {
+      "X-Header-Test": {
+        "name": "X-Header-Test",
+        "in": "header",
```

Comment:
Thanks for the PR!

We can't specify `name` and `in` in Header Object. 
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#header-object

> `name` MUST NOT be specified, it is given in the corresponding headers map.
> `in` MUST NOT be specified, it is implicitly in header.